dev-bins/.config/nextest.toml

```diff
@@ -0,0 +1,17 @@
+[store]
+dir = "target/nextest"
+
+[test-groups]
+build-sbf = { max-threads = 1 }
+
+[profile.ci]
+failure-output = "immediate-final"
+slow-timeout = { period = "60s", terminate-after = 1 }
+retries = { backoff = "fixed", count = 3, delay = "1s" }
+
+[profile.ci.junit]
+path = "../junit.xml"
+
+[[profile.ci.overrides]]
```

Comment:
@yihau did i do this right? i was getting an error about bench-tps not existing in the top-level config. moved it down here + what looked like general settings :shrug:

Reply:
this one:

https://github.com/anza-xyz/agave/pull/8616/commits/7af9fb0a5d0b6ea566b17e7f67dff6a89167bd71

the step: https://buildkite.com/anza/agave/builds/32516#019a0c49-01db-4ce9-bd17-a58b4660e9a8

Reply:
ah! ok i assumed we `./scripts/cargo-for-all-lock-files.sh`ed that somewhere. so if i add that step, i need to move the nextest config back to top level or i leave it in the dev-bins subdir?